frontend/src/components/Trigger.tsx

```diff
@@ -300,8 +301,9 @@ export default class Trigger extends React.Component<TriggerProps, TriggerState>
               variant='outlined'
             >
               {Object.keys(PeriodicInterval).map((interval, i) => (
-                <MenuItem key={i} value={PeriodicInterval[interval]}>
-                  {PeriodicInterval[interval] + (type === TriggerType.INTERVALED ? 's' : '')}
+                 <MenuItem key={i} value={PeriodicInterval[interval]}>
+                  {/* Add "s" based on interval */}
+                  {t(PeriodicInterval[interval])+('(s)')}
```

Comment:
Month in French will show as "Mois(s)"

Reply:
I think the plural concept doesn't works because  even if we choose 1 week in the  'real site' h[ttps://kubeflow.covid.cloud.statcan.ca/_/pipeline/#/runs/new?experimentId= ](url) it put a '  s ' . So I suggest always have a s in that enum

Reply:
![image](https://user-images.githubusercontent.com/58990753/108404367-c1f24500-71ed-11eb-80cf-1634cd3de96e.png)